test/t_web/channels/feed_channel_test.exs

```diff
@@ -311,22 +307,19 @@ defmodule TWeb.FeedChannelTest do
 
       assert feed == [
                %{
-                 "distance" => 9510,
-                 "profile" => %{
-                   user_id: m3.id,
-                   name: "mate-3",
-                   gender: "M",
-                   story: [
-                     %{
-                       "background" => %{
-                         "proxy" =>
-                           "https://d1234.cloudfront.net/1hPLj5rf4QOwpxjzZB_S-X9SsrQMj0cayJcOCmnvXz4/fit/1000/0/sm/0/aHR0cHM6Ly9wcmV0ZW5kLXRoaXMtaXMtcmVhbC5zMy5hbWF6b25hd3MuY29tL3Rlc3Q",
-                         "s3_key" => "test"
-                       },
-                       "labels" => []
-                     }
-                   ]
-                 }
+                 user_id: m3.id,
+                 name: "mate-3",
+                 gender: "M",
+                 story: [
+                   %{
+                     "background" => %{
+                       "proxy" =>
+                         "https://d1234.cloudfront.net/1hPLj5rf4QOwpxjzZB_S-X9SsrQMj0cayJcOCmnvXz4/fit/1000/0/sm/0/aHR0cHM6Ly9wcmV0ZW5kLXRoaXMtaXMtcmVhbC5zMy5hbWF6b25hd3MuY29tL3Rlc3Q",
+                       "s3_key" => "test"
+                     },
+                     "labels" => []
+                   }
+                 ]
```

Comment:
this change {distance: ..., profile: ...}  -> profile {...} does not affect frontend: https://github.com/getsince/test5/blob/rails-fork/Since/Feed/Models/FeedProfile.swift#L41

Reply:
Do you mean `{distance: 1000, profile: {}}` vs `{profile: {}}`?

Reply:
nope, `{distance: 1000, profile: {}}` vs `{}`

Reply:
then frontend is affected